pom.xml

```diff
@@ -660,12 +660,6 @@ flexible messaging model and an intuitive client API.</description>
         <version>${netty-iouring.version}</version>
         <classifier>linux-x86_64</classifier>
       </dependency>
-      <dependency>
-        <groupId>io.netty.incubator</groupId>
-        <artifactId>netty-incubator-transport-native-io_uring</artifactId>
-        <version>${netty-iouring.version}</version>
-        <classifier>linux-aarch_64</classifier>
-      </dependency>
```

Comment:
Why remove this?

Reply:
CI failed with UnsatisifiedLinkError or NoClassDefinedError. I'm debugging now.

The reason for removing aarch64 classifier is that we don't support aarch64 native transport for other alternatives so it can be a different round for debugging.